processor/probabilisticsamplerprocessor/README.md

```diff
@@ -35,6 +35,51 @@ processors:
     sampling_percentage: 15.3
 ```
 
+The probabilistic sampler supports sampling logs according to their trace ID, or by a specific log record attribute.
```

Comment:
This looks very much like the one for traces. Perhaps this readme should be split into sections, one for the common characteristics, one for traces, and one for logs.